### PR TITLE
Feat/form

### PR DESCRIPTION
**Pull request by vmarkwardt**
:-)

### CODE REVIEW

File: src/certificateForm/CertificateFormPage.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { H2 } from '../commons/headings'
+import FormCertificate from './FormCertificate'
+
+export default function ChartPage({ title, categoryList, onProgressChange }) {
```
Comment (melmare) [inline src/certificateForm/CertificateFormPage.js]: Right now it looks like you are not using the props categoryList and onProgressChange. If it is link that, you don't need to destructure them.

File: src/certificateForm/FormCertificate.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import { getISODate } from '../utils'
+
+export default function FormCertificate() {
+  const today = getISODate(new Date())
+  const [certificate, setCertificate] = useState({})
+
+  function handleTitleOnChange(event) {
```
Comment (melmare) [inline src/certificateForm/FormCertificate.js]: Is there maybe a possibility to write an higher order function? In that case you would not need to have 3x the same function with just one word changing.

File: src/certificateForm/FormCertificate.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      <label>
+        Bemerkung:
+        <textarea
+          name="comment"
+          rows="5"
+          cols="33"
+          onChange={handleCommentOnChange}
+        />
+      </label>
+      <button type="submit" onClick={handleOnSubmit} />
```
Comment (melmare) [inline src/certificateForm/FormCertificate.js]: You can also set the onSubmit function on the Form-Element, so the user do not need to push the button and there is also the possibility to submit the formular by pressing a enter key.

File: src/certificateForm/CertificateFormPage.js (hunk 1 above)
Comment (vmarkwardt) [reply src/certificateForm/CertificateFormPage.js]: ups, copy-paste error ;-)

File: src/certificateForm/FormCertificate.js (hunk 2 above)
Comment (vmarkwardt) [reply src/certificateForm/FormCertificate.js]: Yes, I want to create a Component for that

File: src/certificateForm/FormCertificate.js (hunk 3 above)
Comment (vmarkwardt) [reply src/certificateForm/FormCertificate.js]: good idea

**Pull request merged**